wetr.F90

```diff
@@ -36,16 +39,24 @@ subroutine getwetr(carma, igroup, rh, rdry, rwet, rhopdry, rhopwet, rc, h2o_mass
     real(kind=f), intent(out)            :: rhopwet !! wet radius [cm]
     integer, intent(inout)               :: rc      !! return code, negative indicates failure
     real(kind=f), intent(in), optional   :: h2o_mass!! water vapor mass concentration (g/cm3)
-    real(kind=f), intent(in), optional   :: h2o_vp  !! water eq. vaper pressure (dynes/cm2)    
+    real(kind=f), intent(in), optional   :: h2o_vp  !! water eq. vaper pressure (dynes/cm2)
     real(kind=f), intent(in), optional   :: temp    !! temperature [K]
-  
+    real(kind=f), intent(in), optional   :: kappa   !! hygroscopicity parameter (Petters & Kreidenweis, ACP, 2007)
+    type(carmastate_type), intent(in),optional :: cstate  !! the carma state object
+    integer, intent(in),optional                  :: iz      !! level index
+
     ! Local declarations
     real(kind=f)            :: humidity
     real(kind=f)            :: r_ratio
     real(kind=f)            :: wtpkelv, den1, den2, drho_dwt
     real(kind=f)            :: sigkelv, sig1, sig2, dsigma_dwt
     real(kind=f)            :: rkelvinH2O_a, rkelvinH2O_b, rkelvinH2O, h2o_kelv
-        
+    real(kind=f)            :: rvap,rh190
+
```

Comment:
@zywshoon  Here we want a comment to describe the particular parameterization of vapor pressure.

Reply:
@fvitt 
! pvap190h2o uses the vaporp_h2o_murphy2005 equation to calculate the vapor pressure at 190 K for liquid water

Reply:
The comment was added in the code